react/react-training/src/scss/variables/_colors.scss

```diff
@@ -0,0 +1,3 @@
+$alto: #ddd;
```

Comment:
The colors in hex should be in uppercase for consistency amongst projects, check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#colors)